gemhardware/include/gem/hw/glib/GLIBManager.h

```diff
@@ -77,36 +75,25 @@ namespace gem {
           void     createGLIBInfoSpaceItems(is_toolbox_ptr is_glib, glib_shared_ptr glib);
           uint16_t m_amcEnableMask;
 
-          toolbox::task::WorkLoop *p_amc_wl; ///< paralelize the calls to different AMCs
-          toolbox::BSem m_amc_wl_semaphore[MAX_AMCS_PER_CRATE]; // do we need a semaphore for the workloop or each of them?
+          toolbox::task::WorkLoop *p_amc_wl;                     ///< paralelize the calls to different AMCs
```

Comment:
* Naming convention
* Why is a raw pointer needed?

Reply:
At some point, it was not trivial to get smart pointers around some `xdaq` types, which is probably why this remained a raw pointer